src/chains/config/fantom.config.ts

```diff
@@ -54,4 +54,22 @@ export const fantomSetts: VaultDefinition[] = [
     state: VaultState.Guarded,
     protocol: Protocol.Solidly,
   },
+  {
+    name: 'WFTM/SEX',
+    createdBlock: 32237088,
+    depositToken: TOKENS.SMM_WFTM_SEX,
+    vaultToken: TOKENS.BSMM_WFTM_SEX,
+    stage: Stage.Staging,
+    state: VaultState.Guarded,
+    protocol: Protocol.Solidly,
```

Comment:
Shouldn't this be Solidex (In the same way we have Convex vaults instead of Curve).